src/Components/Header.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import {Heading, Center, Text} from '@chakra-ui/react'
+function Header(props: any) {
+    return (
+        <Heading className = "header">
+            <Center className = "player-stats score" >
+            <Text className= 'purple' >{props.p1Score } </Text> <Text color= 'gray' ml = {5} mr = {5}> - </Text> <Text className = 'blue'> { props.p2Score}</Text>
+             
+            </Center>
+            {!props.won? 
```

Comment:
I think props.won just corresponds to whether or not the game was finished. So it would be true even if there was a tie. I know the name is confusing, so I renamed the prop to gameFinished.

Reply:
If it was still a tie and props.player was "/" the element would render "Player 2" as per your conditional right? Wouldn't you want to render something else - like a tie was present if that was the case?